documentation-site/components/yard/config/progress-steps.ts

```diff
@@ -0,0 +1,71 @@
+import {ProgressSteps, Step} from 'baseui/progress-steps';
+import {Button} from 'baseui/button';
+import {PropTypes} from '../const';
+import {TConfig} from '../types';
+
+const ProgressStepsConfig: TConfig = {
+  imports: {
+    'baseui/progress-steps': {named: ['ProgressSteps']},
+  },
+  scope: {
+    ProgressSteps,
+    Step,
+    Button,
+  },
+  theme: [''],
+  props: {
+    current: {
+      value: '0',
+      type: PropTypes.Number,
+      description: 'Defines the current active step index.',
+      stateful: true,
+    },
+    children: {
+      value: `<Step title="Verify Address">
+  <p>Address on file: 1455 Market Street</p>
+  <Button size="compact" onClick={() => setCurrent(1)}>
```

Comment:
in this case, can I update the `current` prop with this value? @tajo

Reply:
sure, this works right? (waiting for the build)

Reply:
Oh I see, it doesn't update the snippet / knob. You could probably try

```jsx
<Button size="compact" onClick={() => {
  setCurrent(1);
  __yard_onChange('ProgressSteps', 'current', 1);
}>
```

It should be taken out (not visible in the code snippet).

Reply:
Um, but that would probably work only initially... we could probably enhance the whole `propHook` thing to target arbitrary inner callbacks.